### DISCUSSION TITLE
v1.0.5

### DISCUSSION THREAD

**Discussion by monkeymademe** [monkeymademe/CamUI#107] [Announcements]
**Fixed Issues** 

- Fixed info/spelling mistakes in Readme: Thanks https://github.com/TimPGamer
- Adjusted the buffer to prevent running out of memory. Every frame written overwrites the last frame
- Tidied up logging
- Fixed an issue that meant non pi cameras where totally skipped in the config building process rendering them hidden.
- Fixed a typo in code for saving config

**New features** 

- Added the ability to switch form Jpeg and MJPeg encoding (H264 is still unavailable as its complicated to implement)
- Added info buttons for capture resolution
- Added Dark/Light Mode switcher 
- Added Version number in the About page: The header 'About Picamera2 WebUI - Verison: XXXX'
- Merged request from @AlexHuebi adding IP as command line variable so users can define their IP instead of having wide open 

<hr /><em>This discussion was created from the release <a href='https://github.com/monkeymademe/picamera2-WebUI/releases/tag/v1.0.5'>v1.0.5</a>.</em>

**Comment by codemonkey2k5** [comment]
What is the easiest way to upgrade to this version?
